responsebot/responsebot_stream.py

```
@@ -50,7 +49,7 @@ def start(self, retry_limit=None):
             except TweepError as e:
                 if 'Stream object already connected!' in e.reason or \
                         'Wrong number of locations points' in e.reason:
-                    raise APIError(e.reason)
+                    raise
```

This code will silently continue without logging any error message at all if the `TweepError` isn't one of the 2 errors being checked by the `if`, won't it?

To be fair this problem is present in the old code too. Even if continuing is really the intended behavior, I guess it's better to at least log the error out. Maybe something like:

```python
else
    logging.error('Skipping API error: %s' % str(e))
```

---

I'm sorry, we should remove this except clause entirely. We only want to raise the `TweepError` whatever it's reason was.